cmake/FindSQLite3.cmake

```diff
@@ -59,5 +59,8 @@ endif()
 
 # Workaround for https://github.com/gazebosim/gz-cmake/issues/312
 if(TARGET SQLite3::SQLite3 AND NOT TARGET SQLite::SQLite3)
-  add_library(SQLite::SQLite3 ALIAS SQLite3::SQLite3)
+  add_library(SQLite::SQLite3 INTERFACE)
```

Comment:
Sorry, my suggestion might have been ill-formed. I don't think we need line 62.

Reply:
Fixed!